service/routes/api.js

```diff
@@ -84,6 +84,22 @@ router.get(/^\/v2\/polyfill(\.\w+)(\.\w+)?/, (req, res) => {
 
 });
 
+router.get(/^\/v2\/detect(\.\w+)(\.\w+)?/, (req, res) => {
+	const polyfills = PolyfillSet.fromQueryParam(req.query.features);
+
+	const params = {
+		features: polyfills.get(),
+		callback: req.query.callback
```

Comment:
Callback is needed for this feature to work, right?  May be worth considering what happens if its missing.